### PR TITLE
adding hf stable diffusion test

### PR DESCRIPTION
**Pull request by sshahrokhi**
# Description

adding a huggingface stable diffusion test.

# Tests

I ran it locally and also oneshot: http://shortn/_T6Q60qNUHl 

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [X] I have performed a self-review of my code.
- [X] I have necessary comments in my code, particularly in hard-to-understand areas.
- [X] I have run one-shot tests and provided workload links above if applicable. 
- [X] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: tests/jax/common.libsonnet [hunk 1]
```diff
@@ -197,4 +197,32 @@ local tpus = import 'templates/tpus.libsonnet';
       |||,
     },
   },
+  huggingFaceDiffuser:: {
```
Comment (RissyRan) [inline tests/jax/common.libsonnet:200]: Thanks for adding this! Do you mind updating `huggingFace` in `huggingFaceTransformers` to distinguish `huggingFaceDiffuser`?

File: tests/jax/latest/flax-sd-pokemon.libsonnet [hunk 2]
```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 Google LLC
```
Comment (RissyRan) [inline tests/jax/latest/flax-sd-pokemon.libsonnet:1]: You may want to update it to `Copyright 2023`

File: tests/jax/latest/flax-sd-pokemon.libsonnet [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  local v4_8 = self.v4_8,
+  v4_8:: v4 {
+    accelerator: tpus.v4_8,
+  },
+
+  local func_tests = [
+    hf_sd_common + func + v4_8,
+  ],
+
+  local conv_tests = [
```
Comment (RissyRan) [inline tests/jax/latest/flax-sd-pokemon.libsonnet:91]: Could we update the type to v4_32 for convergence tests due to better performance?

**Review by RissyRan — COMMENTED** [review]
Thanks! LGTM! Just have a few comments.

**Comment by sshahrokhi** [comment]
Thanks Ran, I agree that hfTransformer makes more sense now :)

**Review by RissyRan — APPROVED** [review]
Thank you!

**Pull request merged**